src/test/java/com/rtb/projectmanagementtool/task/TaskTest.java

```diff
@@ -0,0 +1,259 @@
+package com.rtb.projectmanagementtool.task;
+
+import static com.google.appengine.api.datastore.FetchOptions.Builder.withLimit;
+
+import com.google.appengine.api.datastore.*;
+import com.google.appengine.api.datastore.Query.*;
+import com.google.appengine.api.datastore.dev.LocalDatastoreService;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+/** Class testing tasks */
+public class TaskTest {
```

Comment:
nit:  I would suggest separating the TaskData tests from the TaskController tests